tests/.env

```diff
@@ -0,0 +1,10 @@
+DB_DRIVER="mysql"
+DB_SERVER="127.0.0.1"
+DB_USER="user"
+DB_PASSWORD="pass"
+DB_DATABASE="db"
+DB_SCHEMA=""
+DB_TABLE_PREFIX=""
+DB_PORT=""
+SECURITY_KEY="xwoCk1yMuiH5W7LcoQ4Yt9Y17LbvGLnJ"
+DEFAULT_SITE_URL="https://your-site-url.test/" # Set this to the `entryUrl` param in the `codeception.yml` file.
```

Comment:
I'm not sure if this file should exist - Does the main CraftCMS repo include an example `.env` file?  It would be best to follow their pattern, but certainly there should be no values set.

Reply:
I believe it is required as per the docs. 
https://docs.craftcms.com/v3/testing/testing-craft/getting-started.html#env
It can be a dedicated database used solely for the purpose of executing tests or the database used by the cms in which the plugin is installed. 
Tests can be configured in the codeception.yml to use a fresh clean db by running the install migration prior to execution. as described here
https://docs.craftcms.com/v3/testing/framework/config-options.html#config-options

Reply:
In that case, we should remove all of the values from the file and leave the fields in for the end user to populate

Reply:
.env updated